recipe/meta.yaml

```diff
@@ -1,36 +1,63 @@
-{% set cudnn_version = "7.6.5.32-1" %}
-{% set cudnn_version_x86 = "7.6.5.33-1" %}
 {% set cuda_version = cudatoolkit | replace('.*',"") %}
-{% set dev_sha256_ppc = "fdf3d23bfa3ba22bef9e508473ab20affc25d5b84614475d5cbf0b40573ab04b" %}
-{% set sha256_ppc = "378894e78783f6866e714d85afc199c69754da71cfe8312e0163b5397c713cf4" %}
-{% set dev_sha256_x86 = "644d9e4206371764893408bf961afceaa9ee63175c2670fe9f8577bf5397cf7a" %}
-{% set sha256_x86 = "317483416b6516c0fb3d38f103bdd315352625c2e16f75aa37bb2a9323dd34c1" %}
-
+{% set cudnnvars = {
+  "10.2": {
+    "os": "rhel7",
```

Comment:
Note that cuda10.2 is only supported on rhel7 for ppc64le, so we pull the `rhel7` version here for both platforms and `rhel8` below for the reverse reasons (Cuda11 is only supported on rhel8 for ppc64le).

There is more flexibility on x86-64, but we might as well keep these in sync.